### PR TITLE
Initial commit - Functional App

### PR DESCRIPTION
**Pull request by sowbiba**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Initial commit with a fully functionning app. Review feedbacks will be addressed in a separate PR
| Type?             | new feature
| BC breaks?        | N/A
| Deprecations?     | N/A
| Fixed ticket?     | -
| How to test?      | -
| Possible impacts? | -

Next steps will be 

- Address reviews feedbacks
- Update the README
- Talk with Infra team about how to deploy

### CODE REVIEW

File: src/ContentBuilder.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
```
Comment (matks) [inline src/ContentBuilder.php:8]: To be fixed in another PR: it should be MIT

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "prestashop/help.prestashop.com",
```
Comment (matks) [inline composer.json:2]: To be fixed in another PR

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.htaccess
```
Comment (matks) [inline .gitignore:1]: To be fixed in another PR: remove irrelevant items

File: src/Http/GuzzleAdapter.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Help\PrestaShop\Http;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use Psr\Http\Message\ResponseInterface;
+
```
Comment (matks) [inline src/Http/GuzzleAdapter.php:8]: To be fixed in another PR: use strict types

File: views/ga-tracker.html.twig [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<script type="text/javascript">
+    let _gaq = _gaq || [];
+    _gaq.push(["_setAccount", "UA-2753771-15"]);
```
Comment (matks) [inline views/ga-tracker.html.twig:3]: To be fixed in another PR: make it configurable.

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Approved and we merge. PR feedbacks to be fixed in other PRs.

File: src/ContentBuilder.php [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+    /**
+     * @throws SyntaxError
+     * @throws RuntimeError
+     * @throws LoaderError
+     */
+    public function getContent(string $uri, string | null $version): string
+    {
+        try {
+            $pageInfos = $this->pageInfosBuilder->getPageInfosFromUri($uri, $version);
```
Comment (eternoendless) [inline src/ContentBuilder.php:56]: This property doesn't exist

Comment (eternoendless) [reply src/ContentBuilder.php:56]: Oh I see, it's a PHP 8 feature.
